otelx/jaeger.go

```diff
@@ -0,0 +1,85 @@
+package otelx
+
+import (
+	"strconv"
+	"strings"
+
+	"go.opentelemetry.io/contrib/propagators/b3"
+	jaegerPropagator "go.opentelemetry.io/contrib/propagators/jaeger"
+	"go.opentelemetry.io/contrib/samplers/jaegerremote"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/exporters/jaeger"
+	"go.opentelemetry.io/otel/propagation"
+	"go.opentelemetry.io/otel/sdk/resource"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+	semconv "go.opentelemetry.io/otel/semconv/v1.7.0"
+	"go.opentelemetry.io/otel/trace"
+)
+
+func configureHostPort(c Config) (host, port string) {
+	address := c.Providers.Jaeger.LocalAgentAddress
+	splitAddr := strings.Split(address, ":")
```

Comment:
This is a bit problematic with IPv6 addresses. I recommend to instead use `net.SplitHostPort`.